### DISCUSSION TITLE
Nofail score multiplier doesn't count until you actually die

### DISCUSSION THREAD

**Discussion by GDcheerios** [ppy/osu#14114] [Ideas] [answered]
Say you downloaded a new map and it has a high star rating and your not sure if you could beat it or not, so you put on nofail just incase, and say you do beat it, but your score could have been higher because you were using nofail.

**Comment by peppy** [comment, marked as answer]
This has been brought up many times by people who don't think about what they are saying.

There would be no reason to ever *not* enable NoFail if this was the case.
